api/courses.go

```diff
@@ -576,6 +577,21 @@ func createCourse(c *gin.Context) {
 	go tum.FetchCourses()
 }
 
+func deleteCourse(c *gin.Context) {
+	foundContext, exists := c.Get("TUMLiveContext")
+	if !exists {
+		sentry.CaptureException(errors.New("context should exist but doesn't"))
+		c.AbortWithStatus(http.StatusInternalServerError)
+		return
+	}
+
+	tumLiveContext := foundContext.(tools.TUMLiveContext)
+	fmt.Println(*tumLiveContext.Course)
```

Comment:
```suggestion
```

Reply:
This print statement doesn't seem to add much value. But a little
```go
log.WithFields(log.Fields{
	"user": tumLiveContext.User.ID,
	"course": tumLiveContext.Course.ID,
}).Info("Delete Course Called")
```
Would be nice